src/cmds/changelogs/single.js

```diff
@@ -45,7 +45,7 @@ module.exports = class SingleChangelogCommand {
       return Promise.reject(new Error(`No file path provided. Usage \`${config.get('cli')} ${this.usage}\`.`));
     }
 
-    if (filePath.endsWith('.md') === false || !filePath.endsWith('.markdown') === false) {
+    if (!(filePath.endsWith('.md') || filePath.endsWith('.markdown'))) {
```

Comment:
I've had case-sensitive filesystems on my mind this week and I'm wondering if we should support `.MD` and `.Markdown`, or any casing variation of them.